scripts/ucsc/template-build-cp.sh

```diff
@@ -1,4 +1,7 @@
 #!/bin/bash
+
+set -xe
+
 mkdir -p "$PREFIX/bin"
-cp {program_source_dir}/{program} "$PREFIX/bin"
-chmod +x "$PREFIX/bin/{program}"
+cp -f {program_source_dir}/{program} "${{PREFIX}}/bin"
```

Comment:
Is the force needed, or are we masking potential error here?

Reply:
This is an improvement by @mencian to one of my previous PRs.
I could remove it if there is a chance to mask some problem!

Reply:
I'm not sure about the reason, so I would rather remove.